test/NuGet.Tests.Apex/NuGet.Tests.Apex/Fixtures/VisualStudioOperationsFixture.cs

```diff
@@ -61,10 +61,23 @@ internal VisualStudioHostConfiguration VisualStudioHostConfiguration
                     // Use the same environment to avoid elevation
                     _visualStudioHostConfiguration.InheritProcessEnvironment = true;
 
-                    // Launch in the experimental instance of VS
-                    if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("NUGET_TEST_APEX_EXP")))
+                    // If test is being run in VS, "Developer PowerShell" , or "Developer Command Prompt", use the same install of VS.
+                    // But don't override Apex's env vars if they have already been set.
+                    const string vsUnderTestVariableName = "VisualStudio.InstallationUnderTest.Path";
+                    if (Environment.GetEnvironmentVariable(vsUnderTestVariableName) == null)
                     {
-                        _visualStudioHostConfiguration.CommandLineArguments += " /rootSuffix Exp";
+                        string vsInstallDir = Environment.GetEnvironmentVariable("VSAPPIDDIR") ?? Environment.GetEnvironmentVariable("DevEnvDir");
+                        if (!string.IsNullOrEmpty(vsInstallDir))
+                        {
+                            var devenvPath = Path.Combine(vsInstallDir, "devenv.exe");
+                            Environment.SetEnvironmentVariable(vsUnderTestVariableName, devenvPath);
+
+                            const string rootSuffixVariableName = "VisualStudio.InstallationUnderTest.RootSuffix";
```

Comment:
Could name of env var be shorter?

Reply:
No, it's the environment variable name that the Apex framework itself uses (hence why the value is set, but never read/used).